backend/app/services/ci_monitor_service.py

```
@@ -0,0 +1,378 @@
+    async def _send_condition_satisfied_notification(
+        self, condition: CompletionCondition
+    ):
+        """Send notification that condition is satisfied"""
+        await self._send_notification(
+            condition,
+            event="condition.satisfied",
+            description=f"CI passed for {condition.repo_full_name}/{condition.branch_name}",
+        )
+
+    async def _send_subtask_fully_completed_notification(
+        self, condition: CompletionCondition, subtask: Subtask
```

Ruff (0.14.6)

Unused method argument: `subtask`

(ARG002)